src/commands/convert.js

```diff
@@ -2,11 +2,7 @@ const utils = require('../utils');
 const constants = require('../constants');
 
 const convert = (context, appid, location) => {
-  context.line('Welcome to the Zapier Platform! :-D');
```

Comment:
It would be really cool here to clear out the location folder if it exists. I forgot to pass version to the tool, and it converted my vb app as if it was WB. When I ran it again, it just added to folder rather than replace it.

Reply:
good call out. there's another place we check for an empty folder- we can borrow that behavior.